README.md

```diff
@@ -1,69 +1,43 @@
 <p align="center">
 ➡️
 <a href="https://discord.gg/GFrQsGy">Discord</a> |
- <a href="https://github.com/RustScan/RustScan#-full-installation-guide">Installation Guide</a> |
+ <a href="https://github.com/RustScan/RustScan/wiki/Installation-Guide">Installation Guide</a> |
  <a href="https://github.com/RustScan/RustScan#-usage">Usage Guide</a>
  ⬅️
 <br>
 <img src="pictures/rustscan.png" height=400px width=400px>
 </p>
 <p align="center">
-<u><b> Turns a 17 minutes Nmap scan into 19 seconds. </b></u><br> Find all open ports <b>fast</b> with RustScan, automatically pipe them into Nmap.
+<u><b> The Modern Port Scanner. </b></u><br><b>Fast, smart, effective.</b> 
```

Comment:
This is different from the opening that gets shown at startup. Is it on purpose? Shall we change that line in the opening?
`The Modern Day Port Scanner.`

Reply:
Ah, I was trying to decide between the two. Opinions? I think "Modern port scanner" is more sleek?

Reply:
I think `The Modern Port Scanner ` is also fine. I'm ok with it. :)